metadata/metadata-modeshape/src/main/java/com/thinkbiganalytics/metadata/modeshape/feed/JcrFeedProvider.java

```diff
@@ -708,6 +728,66 @@ public void populateInverseFeedDependencies() {
                 .forEach(depFeed -> depFeed.addUsedByFeed(feed1));
         });
     }
+    
+    @Override
+    protected void appendJoins(StringBuilder bldr, String filter) {
+        if (! Strings.isNullOrEmpty(filter)) {
+            bldr.append("JOIN [tba:feedSummary] AS fs ON ISCHILDNODE(fs, e) ");
+            bldr.append("JOIN [tba:categoryDetails] AS cd ON ISCHILDNODE(e, cd) ");
+            bldr.append("JOIN [tba:category] AS c ON ISCHILDNODE(cd, c) ");
+            bldr.append("JOIN [tba:feedData] AS fd ON ISCHILDNODE(fd, e) ");
+//            bldr.append("JOIN [tba:feedTemplate] AS t ON t.[jcr:uuid] = fd.[tba:feedTemplate] ");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)